values.yaml

```diff
@@ -86,6 +86,8 @@ deploy:
   
   nodeSelector: {}
 
+  jmx: enabled
```

Comment:
This should be:

```yaml
deploy:
  ...
  jmx:
    enabled: true | false
```

The template will then need to be updated to `{{ if .Values.deploy.jmx.enabled }}`